test/accel/CMakeLists.txt

```
@@ -79,6 +79,8 @@ foreach(_basename IN ITEMS UserActionIntegration TrackingManagerIntegration)
     OFFLOAD cpu gpu
     RMTYPE serial
   )
+  # Test OpNovice
+  celeritas_add_integration_tests(${_basename} OpNovice run)
```

This line adds a bunch of different combinations, which is useful for testing primary functionality (GPU compatibility, thread safety) but not necessary for running the same type of problem that's already got a test. Replace with:

```suggestion
  # Test OpNovice
  celeritas_add_integration_tests(
    ${_basename} OpNoviceOptical run
    OFFLOAD cpu gpu g4
    RMTYPE ${_optical_rm_type}
  )
```
after moving the `_optical_rm_type` code above this loop.